includes/class-wp-job-manager.php

```diff
@@ -77,6 +77,7 @@ public function __construct() {
 		include_once JOB_MANAGER_PLUGIN_DIR . '/includes/class-wp-job-manager-data-exporter.php';
 		include_once JOB_MANAGER_PLUGIN_DIR . '/includes/class-wp-job-manager-com-api.php';
 		include_once JOB_MANAGER_PLUGIN_DIR . '/includes/promoted-jobs/class-wp-job-manager-promoted-jobs.php';
+		include_once JOB_MANAGER_PLUGIN_DIR . '/includes/class-wp-job-manager-recaptcha.php';
```

Comment:
If we put the class under the namespace `WP_Job_Manager`, we won't need to include it here because it can use the autoload :D

Reply:
done in cb4210dfe6288eb66c8c715897b1f9ec794e855f